app/assets/javascripts/script.js

```diff
@@ -0,0 +1,8 @@
+
+$(document).on('ready turbolinks:load', () => {scrollerSignUp();})
+
+function scrollerSignUp()
+{
+    $('#productscrolldown').on('click', function()
+    { $('#scroller').removeClass('d-none'); })
```

Comment:
Lets fix indentation and also add `{` on the line where `function` is present ie,

```
    $('#productscrolldown').on('click', function() {
      { $('#scroller').removeClass('d-none');
     });